### PR TITLE
Add example to std::process::abort

### PR DESCRIPTION
**Pull request by rap2hpoutre**
This is a first step in order to complete this issue: https://github.com/rust-lang/rust/issues/29370
I submitted this PR with the help of @steveklabnik More info here: https://github.com/rust-lang/rust/issues/29370#issuecomment-290089330

It's my first PR on Rust, I'm learning how to contribute: Should I ping someone? I will post another PR with a more complicated example soon, I prefer send it separately (cause maybe I made some mistakes).

### CODE REVIEW

**Comment by steveklabnik** [comment]
> Should I ping someone?

Nope, as you can see, @rust-highfive has your back 😄 that said, let's assign this to me, not brian:

r? @steveklabnik

> I will post another PR with a more complicated example soon, I prefer send it separately (cause maybe I made some mistakes).

💯

File: src/libstd/process.rs [hunk 1]
```diff
@@ -1056,6 +1056,19 @@ pub fn exit(code: i32) -> ! {
 /// will be run. If a clean shutdown is needed it is recommended to only call
 /// this function at a known point where there are no more destructors left
 /// to run.
+///
+/// # Examples
+/// ```
```
Comment (steveklabnik) [inline src/libstd/process.rs]: could you add another `///` above this line, please?

**Review by steveklabnik — CHANGES_REQUESTED** [review]
one tiny thing, and then this is good to go! 🎊

File: src/libstd/process.rs (hunk 1 above)
Comment (rap2hpoutre) [inline src/libstd/process.rs]: Ok I updated

Comment (rap2hpoutre) [inline src/libstd/process.rs]: Thanks for all the time you spent to help me!

**Review by steveklabnik — APPROVED** [review]

**Comment by steveklabnik** [comment]
Looks great, thanks! Travis found an issue though:

https://travis-ci.org/rust-lang/rust/jobs/216364459#L564-L566

these three lines have trailing whitespace. Can you remove them?

File: src/libstd/process.rs (hunk 1 above)
Comment (steveklabnik) [inline src/libstd/process.rs]: No problem! Very glad to do so.

**Comment by rap2hpoutre** [comment]
@steveklabnik It fails again but I don't know why... Reading the log does not help me: https://api.travis-ci.org/jobs/216373325/log.txt?deansi=true

**Comment by steveklabnik** [comment]
Looks like a spurious failure; I'm going to retry it.

File: src/libstd/process.rs [hunk 2]
```diff
@@ -1056,6 +1056,20 @@ pub fn exit(code: i32) -> ! {
 /// will be run. If a clean shutdown is needed it is recommended to only call
 /// this function at a known point where there are no more destructors left
 /// to run.
+///
+/// # Examples
+///
+/// ```
```
Comment (ollie27) [inline src/libstd/process.rs]: This needs to be changed to ` ```no_run ` so it doesn't try to run this code as a test.

Comment (rap2hpoutre) [inline src/libstd/process.rs]: Ok thanks, done!

**Comment by rap2hpoutre** [comment]
@steveklabnik  It passes now :)

**Comment by steveklabnik** [comment]
@bors: r+ rollup

wonderful, thank you so much!

**Comment by rap2hpoutre** [comment]
Thanks to you I learned a lot with this (really small) PR! I'm ready for better PR now.

**Comment by steveklabnik** [comment]
Awesome :D

**Pull request merged**